docset/winserver2022-ps/addsdeployment/Test-ADDSDomainControllerInstallation.md

```diff
@@ -11,82 +11,121 @@ title: Test-ADDSDomainControllerInstallation
+ [-MoveInfrastructureOperationMasterRoleIfNecessary] [-ReadOnlyReplica]
+ [-NoRebootOnCompletion] [-ReplicationSourceDC <String>] [-SkipAutoConfigureDns]
+ [-SystemKey <SecureString>] [-SysvolPath <String>] [-Force] [<CommonParameters>]
 ```
 
 ### ADDSDomainControllerUseExistingAccount
+
 ```
-Test-ADDSDomainControllerInstallation -DomainName <String> [-SafeModeAdministratorPassword <SecureString>]
- [-ADPrepCredential <PSCredential>] [-ApplicationPartitionsToReplicate <String[]>] [-Credential <PSCredential>]
- [-CriticalReplicationOnly] [-DatabasePath <String>] [-NoDnsOnNetwork] [-InstallationMediaPath <String>]
- [-LogPath <String>] [-NoRebootOnCompletion] [-ReplicationSourceDC <String>] [-SkipAutoConfigureDns]
- [-SystemKey <SecureString>] [-SysvolPath <String>] [-UseExistingAccount] [-Force] [<CommonParameters>]
+Test-ADDSDomainControllerInstallation -DomainName <String>
+ [-SafeModeAdministratorPassword <SecureString>] [-ADPrepCredential <PSCredential>]
+ [-ApplicationPartitionsToReplicate <String[]>] [-Credential <PSCredential>]
+ [-CriticalReplicationOnly] [-DatabasePath <String>] [-NoDnsOnNetwork]
+ [-InstallationMediaPath <String>] [-LogPath <String>] [-NoRebootOnCompletion]
+ [-ReplicationSourceDC <String>] [-SkipAutoConfigureDns] [-SystemKey <SecureString>]
+ [-SysvolPath <String>] [-UseExistingAccount] [-Force] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Test-ADDSDomainControllerInstallation** cmdlet runs those prerequisite checks (only) which would be performed if you were to use the **Install-ADDSDomainController** cmdlet to install a domain controller in Active Directory.
-It differs from using the *WhatIf* parameter with the **Install-ADDSDomainController** cmdlet in that instead of summarizing the changes that would occur during the installation process, this cmdlet actually tests whether those changes are possible given the current environment.
 
-For more information on the scope of these prerequisite checks that the ADDSDeployment module performs when using this cmdlet see the section ADPrep and Prerequisite Checking Architecture in [AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
+The `Test-ADDSDomainControllerInstallation` cmdlet runs those prerequisite checks (only) which would
+be performed if you were to use the `Install-ADDSDomainController` cmdlet to install a domain
+controller in Active Directory. It differs from using the `-WhatIf` parameter with the
+`Install-ADDSDomainController` cmdlet in that instead of summarizing the changes that would occur
+during the installation process, this cmdlet actually tests whether those changes are possible given
+the current environment.
+
+For more information on the scope of these prerequisite checks that the ADDSDeployment module
+performs when using this cmdlet see the section ADPrep and Prerequisite Checking Architecture in
+[AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
```

Comment:
```suggestion
be performed if you were to use the `Install-ADDSDomainController` cmdlet to install a domain
controller in Active Directory. It differs from using the **WhatIf** parameter with the
`Install-ADDSDomainController` cmdlet in that instead of summarizing the changes that would occur
during the installation process, this cmdlet actually tests whether those changes are possible given
the current environment.

For more information on the scope of these prerequisite checks that the **ADDSDeployment** module
performs when using this cmdlet see the section ADPrep and Prerequisite Checking Architecture in
[AD DS Simplified Administration](https://go.microsoft.com/fwlink/?LinkID=237244).
```